script/openqa-check-devel-repo

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+usage() {
+    cat << EOF
+Usage: openqa-check-devel-repo
+Checks whether devel:openQA packages are stable
+
+Options:
+ -h, --help         display this help
+EOF
+    exit "$1"
+}
+
+opts=$(getopt -o h --long help -n 'parse-options' -- "$@") || usage 1
```

Comment:
is `parse-options` common here? or should it rather be the name of the script?

Reply:
Added another commit to fix it in all scripts.